Libraries/pbxbuild/Headers/pbxbuild/Target/Environment.h

```diff
@@ -50,10 +50,6 @@ class Environment {
     std::vector<std::string>                 _architectures;
```

Comment:
Sure, the mirror I used seems to remove old versions as soon as there is an
update

On Wed, Sep 7, 2016 at 4:41 PM Grant Paul <notifications@github.com> wrote:

> In Libraries/pbxbuild/Headers/pbxbuild/Target/Environment.h
> <https://github.com/facebook/xcbuild/pull/114#discussion_r77924689>:
>
> > @@ -50,10 +50,6 @@ class Environment {
> >      std::vector<std::string>                 _architectures;
>
> Travis is failing from libpng install, @marcsalem
> <https://github.com/marcsalem> could you take a look?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/xcbuild/pull/114/files/8c8c9be7a472d0ef40093a9c4e697d049e770f9e#r77924689>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAmoqA2e0hzYV4yhZOdPKnpGaPe6IDaZks5qn0u-gaJpZM4J0QWk>
> .
>